client/my-sites/email/email-management/home/utils.js

```diff
@@ -181,3 +181,16 @@ export function recordEmailAppLaunchEvent( { app, context, provider } ) {
 		provider,
 	} );
 }
+
+/**
+ * Tracks an event for the key 'calypso_inbox_upsell'.
+ *
+ * @param {app, context, provider} - app/context/provider should be string and must be provided.
+ */
+export function recordInboxUpsellEvent( { context, product, provider } ) {
```

Comment:
I'm wondering if we should make it more explicit that it's recording to Tracks, i.e. `recordInboxUpsellTracksEvent`.